frontends/web/package.json

```diff
@@ -3,16 +3,19 @@
   "name": "bitboxapp",
   "version": "0.0.0",
   "devDependencies": {
-    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
     "@testing-library/jest-dom": "^6.4.6",
     "@testing-library/react": "^16.0.0",
     "@testing-library/user-event": "^14.5.2",
     "@types/node": "^20.12.7",
     "@types/react": "18.3.3",
     "@types/react-dom": "18.3.0",
+    "@typescript-eslint/eslint-plugin": "^7.15.0",
+    "@typescript-eslint/parser": "^7.15.0",
     "@vitejs/plugin-react": "^4.0.4",
     "eslint": "^8.57.0",
-    "eslint-config-react-app": "^7.0.1",
+    "eslint-plugin-react": "^7.34.3",
+    "eslint-plugin-react-hooks": "^4.6.2",
+    "eslint-plugin-react-refresh": "^0.4.7",
```

Comment:
the old plugin contained one more for testinglibrary, could make sense to add that wdyt?

- https://www.npmjs.com/package/eslint-config-react-app?activeTab=dependencies
- https://www.npmjs.com/package/eslint-plugin-testing-library

Reply:
Good find! This might be the reason why we saw the noise on.`make webtest` and needed this https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2801#issuecomment-2217855467.